src/sentry/models/authidentity.py

```diff
@@ -22,6 +27,18 @@ class AuthIdentity(Model):
     last_synced = models.DateTimeField(default=timezone.now)
     date_added = models.DateTimeField(default=timezone.now)
 
+    def outbox_region_names(self) -> Collection[str]:
+        return find_regions_for_orgs([self.auth_provider.organization_id])
```

Comment:
We can configure and customize what regions a control outbox should go to with this override.
By "default" it searches for an organization_id (you can see the imp above) but in cases where a relation is necessary to resolve, we let you implement that by hand.